tests/lib/test_catalog_schema.py

```diff
@@ -61,7 +61,29 @@ def test_has_missing_fields() -> None:
     ]
 
     missing_fields = get_missing_fields(fields)
-    print(missing_fields)
-
     assert len(missing_fields) == 2
     assert missing_fields == set(["licence", "producteur_type"])
+
+
+def test_has_extra_fields() -> None:
+    fields = [
+        "titre",
+        "description",
+        "mots_cles",
+        "nom_orga",
+        "siret_orga",
+        "id_alt_orga",
+        "service",
+        "si",
+        "contact_service",
+        "contact_personne",
+        "date_pub",
+        "date_maj",
+        "freq_maj",
+        "couv_geo",
+        "url",
+        "format",
+        "extra_field_1",
+    ]
+
+    assert get_extra_fields(fields) == set(["extra_field_1"])
```

Comment:
(Nit) On peut écrire directement ceci (ça s'appelle un "set literal", comme `["élément 1", "élément 2"]` est un "list literal") :

```suggestion
    assert get_extra_fields(fields) == {"extra_field_1"}
```